### ISSUE TITLE
JSDoc node

### ISSUE THREAD

**Issue by xiaoxiangmoe** [typescript-eslint/typescript-eslint#1316]
**What code were you trying to parse?**

```ts
const parser = require('@typescript-eslint/typescript-estree');
const code = `/**\n * @type string\n */\nconst hello: string = 'world';`
const ast = parser.parse(code, {
  range: true,
  loc: true,
  })
```

**What did you expect to happen?**

ast.body[0].declarations[0] has jsdoc node

**What actually happened?**

no jsdoc node

**Versions**

| package                                | version |
| -------------------------------------- | ------- |
| `@typescript-eslint/typescript-estree` | `2.10.0` |
| `TypeScript`                           | `3.7.2` |
| `node`                                 | `13.2.0` |
| `npm`                                  | `6.13.1` |

**Comment by bradzacher** [comment]
Comments are included in the output, if you pass the `comments` option to the parser.
